InnaFeature/Pages/AlertsFramesAndWindows.cs

```diff
@@ -0,0 +1,22 @@
+﻿using InnaFeature.Helpers.Browser;
+using OpenQA.Selenium;
+
+namespace InnaFeature.Pages
+{
+    internal class AlertsFramesAndWindows : BasePage
+    {
+        public AlertsFramesAndWindows(IBrowserHelper browserHelper) : base(browserHelper)
+        {
+
+        }
+        public IWebElement NewTabButton =>
+            browserHelper.WebDriver.FindElement(By.XPath("//*[@id = 'tabButton']"));
+        public IWebElement PresentedText =>
+            browserHelper.WebDriver.FindElement(By.XPath("//*[@id = 'sampleHeading' and contains(text(), 'This is a sample page')]"));
```

Comment:
Please add new lines after each IWebElement declaration. Same for all other page files